mono_repo/lib/src/travis_config.dart

```diff
@@ -138,7 +145,12 @@ class TravisJob extends Object with _$TravisJobSerializerMixin {
 
 @JsonSerializable(includeIfNull: false)
 class DartTask extends Object with _$DartTaskSerializerMixin {
-  static final _tasks = const ['dartfmt', 'dartanalyzer', 'test'];
+  static final _tasks = const [
+    'dartfmt',
+    'dartanalyzer',
+    'test',
+    'build_and_test',
```

Comment:
Could we just call this 'custom' or 'shell' or similar?

...make it more general.

Reply:
Oops, this was just something I was playing around with, but didn't end up pursuing. Forgot to clean it all up.